wasm/lib/src/browser_module.rs

```diff
@@ -270,6 +263,65 @@ fn promise_catch(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     Ok(PyPromise::new_obj(promise_type, ret_promise))
 }
 
+#[derive(Debug)]
+struct PyDocument {
```

Comment:
nit: can this just be `Document` (and same for others)? Outside of the core built-in types, the prefixes just seem like noise to me.

Reply:
Sure, for `PyJsValue` I think I'll keep the prefix but for these I'll drop it.